project/frontend/src/components/HeaderBar/HeaderBar.js

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import classes from "./HeaderBar.module.css"
+import Logo from "../../assets/sGonksLogo.png"
+import { Link, NavLink } from "react-router-dom"
+import LinkButton from "../UI/LinkButton/LinkButton"
+import LoginButtonSet from "../UI/LoginButtonSet/LoginButtonSet";
+
+const HeaderBar = (props) => {
+  let innerNavLinks = [
+    { linkTo: "/", display: "My sGonks" },
+    { linkTo: "/", display: "Competition" },
+    { linkTo: "/", display: "Marketplace" },
+  ]
+
+  let innerNav = props.innerNav ? (
```

Comment:
when would props.innerNav  be null?

Reply:
* When the user is not logged in
* When the user is logged in but hasn't selected a competition
Inner nav is for navigation WITHIN a specific competition, so doesn't make sense to have them in the above ^ scenarios.

Reply:
Sorry, just to clairfy. innerNav and props.innerNav are different things. 
props.innerNav is a boolean variable passed into this class.
innerNav is the display of the actual inner navigation items. Which would be empty (null) or populated. 

innerNav displays things if props.innerNav is true.

Reply:
Hopefully that clears things up :)